pom.xml

```diff
@@ -62,6 +62,7 @@
     <maven.build.timestamp.format>yyyyMMdd</maven.build.timestamp.format>
     <build.timestamp>${maven.build.timestamp}</build.timestamp>
     <build.counter>0</build.counter>
+    <maven.tomcat.port>8082</maven.tomcat.port>
```

Comment:
This only affects developer environments where tomcat is run via `mvn tomcat7:run` right? I've been unsure if this property has any effect on our salted machines which seem to have this value set in server.xml instead.

Reply:
Yes, this should only matter when run via maven.